05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,47 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+body {
+
 }
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+section {
+
 }
 
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.left-box {
+  float: left;
+  width: 500px;
 }
 
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+.right-box {
+  background-color: #F0F0F0;
+  vertical-align: middle;
+  text-align: center;
+  float: left;
+  width: 500px;
+  padding-top: 100px;
+  padding-bottom: 100px;
 }
 
-.left h1 {
-    text-align: center;
+h1 {
+  color: #252b3b;
+  font-size: 1.9em;
 }
 
-.left p {
-    text-align: center;
+p {
+  color: #555;
+  padding-bottom: 20px;
 }
 
-body p {
-    font-family: monospace;
-    font-size: 12px;
-    text-decoration: underline;
+img {
+  width: inherit;;
```

Comment:
instead of height and width being inherit, try setting just the width 10 100%. the height will auto fill to maintain proportions of the image.